src/Evolution/ComputeTags.hpp

```diff
@@ -39,6 +39,29 @@ struct AnalyticCompute : ::Tags::AnalyticSolutions<AnalyticFieldsTagList>,
   }
 };
 
+/*!
+ * \brief Use the `AnalyticDataTag` to compute the analytic data for the
+ * tags in `AnalyticFieldsTagList`.
+ */
+template <size_t Dim, typename AnalyticDataTag, typename AnalyticFieldsTagList>
+struct AnalyticDataCompute : ::Tags::AnalyticSolutions<AnalyticFieldsTagList>,
```

Comment:
This tag probably shouldn't derive off `AnalyticSolutions` since the analytic data is no analytic solution. Do you need to keep the analytic data around in the DataBox at all? Do you need it for anything else than retrieving the initial data during initialization?